### PR TITLE
Add net.fsuae.FS-UAE

### PR DESCRIPTION
**Pull request by digitalstreamio**

### CODE REVIEW

File: net.fsuae.FS-UAE.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.fsuae.FS-UAE.desktop</id>
+  <name>FS-UAE</name>
+  <summary>FS-UAE Amiga Emulator</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```
Comment (barthalion) [inline net.fsuae.FS-UAE.appdata.xml]: `CC0-1.0` is preferred.

File: net.fsuae.FS-UAE.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <screenshot>https://fs-uae.net/images/fs-uae/front1.png</screenshot>
+    <screenshot>https://fs-uae.net/images/fs-uae/front2.png</screenshot>
+    <screenshot>https://fs-uae.net/images/fs-uae/front3.png</screenshot>
+    <screenshot>https://fs-uae.net/images/fs-uae/front5.png</screenshot>
+  </screenshots>
+  <releases>
+    <release date="2018-02-24" version="2.8.4"/>
+  </releases>
+  <update_contact>digitalstreamio@gmail.com</update_contact>
+</application>
```
Comment (barthalion) [inline net.fsuae.FS-UAE.appdata.xml]: Please add [OARS](https://hughsie.github.io/oars/) ratings.

File: net.fsuae.FS-UAE.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            "post-install": [
+                "install -Dm644 net.fsuae.FS-UAE.appdata.xml /app/share/appdata/net.fsuae.FS-UAE.appdata.xml",
+                "sed -i s/NoDisplay=.*/NoDisplay=false/ /app/share/applications/fs-uae.desktop",
+                "cp /app/share/mime/packages/fs-uae.xml /app/share/mime/packages/net.fsuae.FS-UAE.xml"
+            ],            
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/FrodeSolheim/fs-uae",
+                    "branch": "stable"
```
Comment (barthalion) [inline net.fsuae.FS-UAE.json]: You should add specific `commit` here so the build is deterministic.

**Comment by barthalion** [comment]
bot, build net.fsuae.FS-UAE

**Comment by barthalion** [comment]
Build status: https://flathub.org/builds/#/builders/1/builds/2271

**Comment by barthalion** [comment]
Would be also nice to invite upstream developer to (co-)maintain it.

File: net.fsuae.FS-UAE.appdata.xml (hunk 1 above)
Comment (digitalstreamio) [reply net.fsuae.FS-UAE.appdata.xml]: Done

File: net.fsuae.FS-UAE.appdata.xml (hunk 2 above)
Comment (digitalstreamio) [reply net.fsuae.FS-UAE.appdata.xml]: Done

File: net.fsuae.FS-UAE.json (hunk 3 above)
Comment (digitalstreamio) [reply net.fsuae.FS-UAE.json]: Done

**Comment by digitalstreamio** [comment]
@barthalion How would I go about adding co-maintainers? Sorry, not familiar with the whole process yet.

**Comment by digitalstreamio** [comment]
FYI - Looks like build bot failed to download one of the dependencies. It looks as sf.net where it is hosted is under DDOS according to their status page.

https://twitter.com/sfnet_ops

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by barthalion** [comment]
> How would I go about adding co-maintainers? Sorry, not familiar with the whole process yet.

That's something we are handling on our end. Author(s) would get push access and bug tracker permissions on the newly created repo so they can handle updates themselves. It's not mandatory but we're aiming at inviting upstream developers when possible.

**Comment by barthalion** [comment]
At least libmpeg2 needs newer `config.sub` to build on arm and aarch64. [This is how](https://github.com/flathub/org.audacityteam.Audacity/blob/f28060d39597a191f241a4dd57abab0f3d017aec/org.audacityteam.Audacity.json#L63) Audacity does it, but you can also run `autoreconf -fi` to be safe (number of manifests from shared-modules do it if you need an example).

**Comment by digitalstreamio** [comment]
Ok. I'll take a look and update PR.

**Comment by digitalstreamio** [comment]
bot, build net.fsuae.FS-UAE

**Comment by digitalstreamio** [comment]
@barthalion Could you kick off another build? Look like I can't quite do that yet on my own. :)

**Comment by TingPing** [comment]
bot, build net.fsuae.FS-UAE

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by digitalstreamio** [comment]
I think we are getting there. Latest changes disable JIT on platforms that do not support it.

**Comment by digitalstreamio** [comment]
Could we do another build?

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by digitalstreamio** [comment]
It looks like build failed again due to SF being down. SourceForge is having a rough month after moving to a new data center. 

https://twitter.com/sfnet_ops

Once they are back operational, I will update the pull and we can try again.

**Comment by digitalstreamio** [comment]
Looks like SF is back online. Could we do another build?

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by digitalstreamio** [comment]
It looks like arm completed successfully and arm64 failed. I have pushed a patch to address the arm64 build issue. Could we try another build?

**Comment by barthalion** [comment]
bot, build net.fsuae.FS-UA

**Comment by digitalstreamio** [comment]
net.fsuae.FS-UA.json is in the repo, build bot issues? 

```
flatpak-builder -v --force-clean --sandbox --delete-build-dirs --user --install-deps-from=flathub --arch x86_64 --bundle-sources --mirror-screenshots-url=https://flathub.org/repo/screenshots --repo repo --extra-sources=/srv/buildbot/worker/build-x86_64/../downloads --default-branch stable --subject 'fix arm64 build (5e33be27)' builddir net.fsuae.FS-UA.json
 in dir /srv/buildbot/worker/build-x86_64/net.fsuae.FS-UA (timeout 1200 secs)
 watching logfiles {}
 argv: ['flatpak-builder', '-v', '--force-clean', '--sandbox', '--delete-build-dirs', '--user', '--install-deps-from=flathub', '--arch', 'x86_64', '--bundle-sources', '--mirror-screenshots-url=https://flathub.org/repo/screenshots', '--repo', 'repo', '--extra-sources=/srv/buildbot/worker/build-x86_64/../downloads', '--default-branch', 'stable', '--subject', 'fix arm64 build (5e33be27)', 'builddir', 'net.fsuae.FS-UA.json']
 using PTY: True
Can't load 'net.fsuae.FS-UA.json': Failed to open file '/srv/buildbot/worker/build-x86_64/net.fsuae.FS-UA/net.fsuae.FS-UA.json': No such file or directory
program finished with exit code 1
elapsedTime=0.007153
```

**Comment by digitalstreamio** [comment]
Looks like build bots weren't happy yesterday, could we do another build today?

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UA

**Comment by digitalstreamio** [comment]
The previous two builds failed as the build bot command was missing a letter at the end of app-id.  It should be 

bot, build net.fsuae.FS-UAE

Could you kick off another build?

**Comment by nedrichards** [comment]
bot, build net.fsuae.FS-UAE

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.fsuae.FS-UAE

**Pull request closed without merging**